src/components/ui/Hero/Hero.tsx

```diff
@@ -0,0 +1,24 @@
+import type { HTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface HeroProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  children: React.ReactNode
```

Comment:
Is there some reason to have this `children` attribute here? Not sure we need it.